policyengine_us/parameters/gov/states/or/tax/income/rates/single.yaml

```diff
@@ -57,6 +60,7 @@ brackets:
         2021-01-01: 9_200
         2022-01-01: 9_450
         2023-01-01: 10_200
+        2024-01-01: 10_750
       metadata:
         uprating: gov.irs.uprating
         rounding:
```

Comment:
```suggestion
      metadata:
        uprating: 
           parameter: gov.irs.uprating
           rounding:
               type:
```

Reply:
Need to check the web app if both work but I thought we need to nest the parameter as well as rounding under the uprating 

![image](https://github.com/user-attachments/assets/367e017c-0a80-47c9-b974-ab7a9d77a90b)

Reply:
I might be wrong here, need to check the web app